automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspacesAzureApiSpec.scala

```diff
@@ -32,7 +41,10 @@ class AzureWorkspacesSpec extends AnyFlatSpec with Matchers with CleanUp {
     Some(UUID.fromString("f41c1a97-179b-4a18-9615-5214d79ba600"))
   )
 
+  private val wsmUrl = Rawls.url.replaceFirst("rawls", "workspace")
```

Comment:
This is certainly a hack, but I think I would have to do the following to get the URL without hacking that we already know about:

1. Add a workspace manager URL to the [conf template file](https://github.com/broadinstitute/firecloud-automated-testing/blob/cfea0225cc3eabb6041552f616af424530af37ee/configs/rawls/application.conf.ctmpl#L14) in `firecloud-automated-testing`.